Networking/Networking/Remote/ReceiptRemote.swift

```diff
@@ -52,16 +52,7 @@ public final class ReceiptRemote: Remote {
 
 extension ReceiptRemote: POSReceiptsRemoteProtocol {
     public func sendPOSReceipt(siteID: Int64, orderID: Int64) async throws {
```

Comment:
I spent more time than needed here: On one end since we use the endpoint just for POS I wanted to make it clear in the function name and wrap it, on other end, there's really no reason to. Updated on 16218bdf24db8c26b5c0db82716e7defd480a27f